test/D2L.SQL.UnitTests/ValidatorTests.cs

```diff
@@ -106,15 +106,10 @@ internal sealed class ValidatorTests {
 					GROUP BY PHOENIX_TABLE.A.USERID
 					HAVING PHOENIX_TABLE.A.USERID = 1234", 
 					TestName = "Compound column names in HAVING")]
-		[TestCase(@"SELECT COURSE_ACCESS_194.A.CONTEXTID, COURSE_ACCESS_194.A.ACTORID, COURSE_ACCESS_194.A.TIMESTAMP, COURSE_ACCESS_194.AGGRID, COURSE_ACCESS_194.A.TYPE, 
-				COURSE_ACCESS_194.A.V, TOOL_ACCESS_BY_ROLE_AND_TOOL_192.A.ROLEID, TOOL_ACCESS_BY_ROLE_AND_TOOL_192.A.TOOLID, TOOL_ACCESS_BY_ROLE_AND_TOOL_192.A.V, 
-				TOOL_ACCESS_BY_ROLE_AND_TOOL_192.A.TIMESTAMP, TOOL_ACCESS_BY_ROLE_AND_TOOL_192.AGGRID
-				FROM COURSE_ACCESS_194 COURSE_ACCESS_194, TOOL_ACCESS_BY_ROLE_AND_TOOL_192 TOOL_ACCESS_BY_ROLE_AND_TOOL_192
-				WHERE COURSE_ACCESS_194.""A.TIMESTAMP"" = TOOL_ACCESS_BY_ROLE_AND_TOOL_192.""A.TIMESTAMP\""", TestName = "AAAAAAAAAAAAAAA")]
-		[TestCase( @"SELECT COURSE_ACCESS_179.A.CONTEXTID, COURSE_ACCESS_179.A.V, COURSE_ACCESS_179.A.TIMESTAMP, COURSE_ACCESS_179.A.ACTORID, COURSE_ACCESS_179.A.TYPE, COURSE_ACCESS_179.AGGRID
-FROM COURSE_ACCESS_179 COURSE_ACCESS_179
-WHERE (COURSE_ACCESS_179.A.CONTEXTID not like '66%')
-ORDER BY COURSE_ACCESS_179.A.TIMESTAMP DESC", TestName = "BBBBBBBBBBBBBB" )]

```

Comment:
Was this reverted in the next commit? If so, any chance we can delete this commit?